components/monitoring/grafana/development/grafana-login.yaml

```diff
@@ -0,0 +1,11 @@
+# After the grafana route is created, wwe are still unable to connect to the Grafana UI
+# because we did not add security to the Grafana yaml file.
+# Instead, use this file after Grafana is installed to set your username/password
+# 1 - edit your desired admin_user and admin_password
```

Comment:
`edit` --> Edit

Reply:
I removed this file and added the explanation to the README file